bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/DefaultEntryLogger.java

```diff
@@ -1036,6 +1036,9 @@ public EntryLogMetadata getEntryLogMetadata(long entryLogId, AbstractLogCompacto
         // entry log
         try {
             return extractEntryLogMetadataFromIndex(entryLogId);
+        } catch (FileNotFoundException ex) {
+            LOG.warn("Cannot find entry log file {}.log", entryLogId);
+            throw new IOException(ex);
```

Comment:
Why wrap the FileNotFoundExpcetion.

Reply:
Please read the context above. @horizonzy